Makefile

```diff
@@ -229,17 +269,28 @@ find:
 	$(eval include global_config/find-domain.sh)
 
 set-azure-account:
-	echo "Logging on to ${AZURE_SUBSCRIPTION}"
-	az account set -s $(AZURE_SUBSCRIPTION)
+	echo "Logging on to ${AZ_SUBSCRIPTION}"
+	az account set -s $(AZ_SUBSCRIPTION)
 
 set-azure-resource-group-tags: ##Tags that will be added to resource group on its creation in ARM template
 	$(eval RG_TAGS=$(shell echo '{"Portfolio": "Early Years and Schools Group", "Parent Business":"Teacher Training and Qualifications", "Product" : "Find postgraduate teacher training", "Service Line": "Teaching Workforce", "Service": "Teacher services", "Service Offering": "Find postgraduate teacher training", "Environment" : "$(ENV_TAG)"}' | jq . ))
 
 set-azure-template-tag:
 	$(eval ARM_TEMPLATE_TAG=1.1.0)
 
+base-resources: set-azure-account set-azure-template-tag set-azure-resource-group-tags
+	az deployment sub create --name "resourcedeploy-tsc-$(shell date +%Y%m%d%H%M%S)" \
+		-l "UK South" --template-uri "https://raw.githubusercontent.com/DFE-Digital/tra-shared-services/${ARM_TEMPLATE_TAG}/azure/resourcedeploy.json" \
+		--parameters "resourceGroupName=${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-rg" 'tags=${RG_TAGS}' \
+			"tfStorageAccountName=${AZURE_RESOURCE_PREFIX}${SERVICE_SHORT}tfstate${CONFIG_SHORT}sa" "tfStorageContainerName=${SERVICE_SHORT}-tfstate" \
+			"keyVaultName=${AZURE_RESOURCE_PREFIX}-${SERVICE_SHORT}-${CONFIG_SHORT}-kv" ${WHAT_IF}
+
+validate-base-resources: set-what-if base-resources
+
+deploy-base-resources: check-auto-approve base-resources
```

Comment:
Could we standardise on "deploy-arm-resources" and "arm-deployment"? As in https://github.com/DFE-Digital/teacher-services-hedgedoc/blob/main/Makefile#L49